### PR TITLE
whitelist of files for npm package

### PR DESCRIPTION
**Pull request by fanatid**
Only src, because (from docs):

>
Certain files are always included, regardless of settings:
  * package.json
  * README (and its variants)
  * CHANGELOG (and its variants)
  * LICENSE / LICENCE

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,21 @@
+sudo: false
+os:
+  - linux
```
Comment (dcousens) [inline .travis.yml]: why `linux`?
I mean,  I guess I always assumed this was the case,  but,  wouldn't it be default?

Comment (fanatid) [inline .travis.yml]: Good question, `linux` is os by default, but I can't find this in the travis docs. Should I remove this line?

Comment (dcousens) [inline .travis.yml]: I think its overly verbose and doesn't serve any meaningful purpose.  Personally.

**Comment by fanatid** [comment]
updated
@dcousens please check slack

**Comment by dcousens** [comment]
Cheers @fanatid,  great work :)

**Pull request merged**